network/src/connectivity_manager/mod.rs

```diff
@@ -439,39 +450,25 @@ where
             // Create future which completes by either dialing after calculated
             // delay or on cancellation.
             let f = async move {
-                let delay = f_delay.deadline().duration_since(now);
-                debug!(
-                    NetworkSchema::new(&network_context)
-                        .remote_peer(&peer_id)
-                        .network_address(&addr),
-                    delay = delay,
-                    "{} Dial future started {} at {}",
-                    network_context,
-                    peer_id,
-                    addr
-                );
```

Comment:
Okay, finally no "log when created" and "log when started"

Reply:
this was getting on my nerves too 😅